src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,119 @@
+package subway;
+
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.jdbc.Sql;
+import static org.springframework.test.annotation.DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD;
+import static subway.helper.LineTestAssuredHelper.*;
+import static subway.helper.LineTestRequestHelper.*;
+
+@DisplayName("지하철노선 관련 기능")
+@DirtiesContext(classMode = AFTER_EACH_TEST_METHOD)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@Sql(scripts = {"classpath:data/station-init.sql"})
+public class LineAcceptanceTest {
+
+    final String 신분당선 = "신분당선";
+    final String 분당선 = "분당선";
+    final String 빨간색 = "bg-red-600";
+    final String 노란색 = "bg-yellow-600";
+    final Long 청계산입구역_ID = 1L;
+    final Long 판교역_ID = 2L;
+    final Long 선릉역_ID = 3L;
+
+    /**
+     * When 지하철 노선을 생성하면
+     * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+     */
+    @DisplayName("지하철노선 생성")
+    @Test
+    public void createStationLine(){
+        //when
+        지하철노선_생성(신분당선, 빨간색, 청계산입구역_ID, 판교역_ID, 10);
+        //then
+        ExtractableResponse<Response> getResponse = 지하철노선_조회();
```

Comment:
```suggestion
        var lineResponse = 지하철노선_조회();
```

가독성을 위하여 타입 추론을 사용하는 것도 좋아보여요!